### ISSUE TITLE
how can i get reported result on voc MIoU 82.9?

### ISSUE THREAD

**Issue by oujieww** [zhanghang1989/PyTorch-Encoding#111]
i train to train encent with resnet101 with auxloss , train  on train.txt10582+val.txt1449 and run test.txt ,only get MIoU 79.80 from voc test server 。i use the model resnet101 download from your setting，with epoch=80 aux_weight=0.2 ，how can i get reported 82.9 ,please help me

**Comment by my462** [comment]
You need to get results on the test set and submit it to the voc server to get the score

**Comment by oujieww** [comment]
@my462 yeah，i run my trained model on test set and get result from test server ，get miou 79.80

**Comment by zhanghang1989** [comment]
Pre-train on Pascal_AUG (augmented set) and then fine-tune on Pascal_VOC.
For the best result COCO pre-training => Pascal AUG => Pascal VOC

**Comment by oujieww** [comment]
@zhanghang1989
thank you

**Comment by gaowu9595** [comment]
@oujieww Hi, I also train the model using context encoding on pascal voc2012 dataset with command 

# First finetuning COCO dataset pretrained model on augmented set
# You can also train from scratch on COCO by yourself
python train.py --batch-size 4 --dataset Pascal_voc --model encnet --aux  --se-loss --backbone resnet101 --lr 0.0001 --checkname res101 --resume runs/Pascal_aug/encnet/res101/model_best.pth.tar --ft
# Finetuning on original set
python train.py --batch-size 4 --dataset pascal_voc --model encnet --aux  --se-loss --backbone resnet101 --lr 0.0001 --checkname res101 --resume /runs/Pascal_voc/encnet/res101/model_best.pth.tar --ft

but only got **MIoU 75.697** from voc test server, I just modify the batch size from 16 to 4 because I only have 2 GPUs(1080ti), How do you train the model to gain MIoU 79.80 , can you tell me some details about it?

**Comment by zhanghang1989** [comment]
Batch size of 4 will make a huge difference.
